aten/src/ATen/native/nested/NestedTensorUtils.cpp

```diff
@@ -87,14 +87,12 @@ std::vector<Tensor> chunk_nested_tensor(const Tensor& self, int64_t chunks, int6
       // This copys offsets so we are safe to move
       auto new_offsets = std::vector<int64_t>(offsets);
       int64_t *size_ptr = new_sizes.data_ptr<int64_t>();
-      int64_t *stride_ptr = new_strides.data_ptr<int64_t>();
       // Get start val for each split
       int64_t start_val = split_idx * split_size;
       for (int64_t i : c10::irange(n_tensors)) {
         const int64_t index = i * tensor_dim + dim;
-        new_offsets[i] = offsets[i] + start_val * stride_ptr[index];
```

Comment:
What happened here?

Reply:
Since we check that the input is contiguous  stride_ptr[index] which is the stride of the last dim in n_tensor[i] will always equal 1. Was working through this when going over the strides calc for as_strided. I can undo but it is the same resulting calc